src/lib/hooks/useUserGeoplocation.tsx

```diff
@@ -0,0 +1,69 @@
+import { useCallback, useEffect, useState } from 'react'
+
+const LOCAL_STORAGE_KEY = 'useGeolocation'
+
+const requestUserGeolocation = (): Promise<GeolocationPosition> =>
+  new Promise((resolve, reject) => {
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(resolve, reject)
+    } else {
+      reject()
+    }
+  })
+
+export const useUserGeoplocation = (): {
```

Comment:
Just a typo: The `p` that is not supposed to be there.

Reply:
Fixed!